docs/python.org/CVE-Numbering-Authority.md

```diff
@@ -0,0 +1,151 @@
+"Public at" field until the advisory is published.
+
+Refer to [CVE Record Requirements](https://www.cve.org/ResourcesSupport/AllResources/CNARules#section_8_cve_record_requirements) in the CNA Rules for a full list of requirements and recommendations.
+
+* [Sign in to Vulnogram](#signing-in-to-vulnogram).
+* In the CVE Portal tab, select the "Reserved" filter option for CVE IDs.
+* Find the CVE ID corresponding to the vulnerability you'd like to publish.
+* Click the CVE ID link to open the CVE Record editor.
+* Fill in the following fields:
+  * Title
+  * Public at (corresponding to the publish date/time of the advisory)
+  * Problem Type (Use [CWE Database](https://cwe.mitre.org/) for help finding CWE IDs)
+  * Vendor or project (Use `Python Software Foundation` or `Pip maintainers`)
+  * Product name (Use `CPython` or `pip`)
+  * Source repository (`https://github.com/python/cpython`)
+  * Versions (double check these are accurate by looking in the "Preview" tab)
+  * CVE Description
+    * The CVE Description has specific requirements like affected version and the impact.
+    * See [Prose Description Requirements](https://www.cve.org/ResourcesSupport/AllResources/CNARules#section_8-2_cve_record_prose_description_requirements) in the CNA Rules for more information.
+  * Credits
+    * Credits of type `reporter`, `coordinator`, `remediation developer`, and `remediation reviewer` are recommended.
+    * Reporter may want to remain anonymous, we should respect this request. Other roles are likely public information in GitHub.
+  * References with the following tags
+    * Link to the advisory on `security-announce@python.org` archive with tag `vendor-advisory`
+    * Link to GitHub pull request with tag `patch`
+    * Link to GitHub issue with tag `issue-tracking`
+* If applicable, fill in the following fields:
+  * Platforms (if vuln only affects a subset of OSes/architectures/bit depths)
+  * Package collection URL (`https://pypi.org` for pip)
+  * Package name (`pip`)
+  * Modules, components, features
+* If no CVSS score has been assigned, ensure that the CVSS score metric section has been explicitly removed from the record.
+  By default, this section is a 10/10 CVSS score.
+* After filling in all fields, check everything looks accurate in the "Preview" tab.
+* After filling in all fields, send the CVE Record JSON to another CNA operator for review:
+  * In the "Source" tab of Vulnogram copy the JSON document.
+  * Paste the JSON document into the `#cna` PSF Slack channel and ask for a review.
+  * Reviewers can paste the CVE Record JSON into their own Vulnogram instance to review the fields.
+* Once approved, the CVE Record is ready to be published. Select `Post to CVE.org` at the bottom of the Editor tab to immediately publish the CVE Record.
+
+### CVE ID Assignment Requests
+
+We'll receive requests from the `cna at python.org` mailing list requesting CVE IDs.
```

Comment:
Do we have any requirements as far as responding to the requestor like with disputes/updates?

Reply:
I've asked in the CNA Slack channel, it's not clear from the rules.

Reply:
Got one reply back in the CNA channel which seems to indicate there's no timeline around CVE ID issuance (and that the timeliness rules for CVE Disputes isn't even documented in the rules). We should be good to go with this for now, then once the rules are updated to make everything clear we can update our processes.